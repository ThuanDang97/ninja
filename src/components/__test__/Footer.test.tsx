import { render, screen } from '@testing-library/react'
import Footer from '@/components/Footer'

describe('Footer', () => {
  it('renders Footer toMatchSnapshot', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
