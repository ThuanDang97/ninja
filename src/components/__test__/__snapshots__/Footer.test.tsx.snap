// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer toMatchSnapshot 1`] = `
<div>
  <footer>
    Copyright 2022 Ninja List
  </footer>
</div>
`;
